commercetools/resource_category.go

```diff
@@ -0,0 +1,284 @@
+package commercetools
+
+import (
+	"context"
+	"log"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/labd/commercetools-go-sdk/commercetools"
+)
+
+func resourceCategory() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceCategoryCreate,
+		Read:   resourceCategoryRead,
+		Update: resourceCategoryUpdate,
+		Delete: resourceCategoryDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+		Schema: map[string]*schema.Schema{
+			"key": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+			},
+			"name": {
+				Type:     schema.TypeMap,
```

Comment:
There's a 
`const TypeLocalizedString = schema.TypeMap` 
type alias in utils.go which can be used for documentation purposes and clarity in the case of Maps which are Localized Strings. You can use that here (and for the other localized string fields).